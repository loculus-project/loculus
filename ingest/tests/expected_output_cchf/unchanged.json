{"KX014000.1.M": "LOC_0000CC1", "KX014001.1.S": "LOC_0000CC2"}