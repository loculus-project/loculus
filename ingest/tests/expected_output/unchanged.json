{
    "KX013462.1.L/KX013463.1.M/KX013464.1.S": "LOC_0000VF9"
}